src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/UnusedLocalVariableCheck.java

```diff
@@ -749,7 +749,7 @@ private static boolean isStandAloneIncrementOrDecrement(DetailAST identAst) {
      */
     private static boolean isIncrementOrDecrementVariableUsed(DetailAST exprAst) {
         return TokenUtil.isOfType(exprAst.getParent(),
-                TokenTypes.ELIST, TokenTypes.INDEX_OP, TokenTypes.ASSIGN)
+                TokenTypes.ELIST, TokenTypes.INDEX_OP, TokenTypes.ASSIGN, TokenTypes.LITERAL_SWITCH)
```

Comment:
Please extract these tokens to a descriptively named array.

Reply:
done